docs/pages/guide/02-start.md

```diff
@@ -0,0 +1,409 @@
+---
+layout: guide
+group: guide
+title: Setup and Installation Guide
+---
+
+* TOC
+{:toc}
+
+
+<span class="c29">Yavin UI browser support</span>
+-------------------------------------------------
+
+<span class="c11">Yavin UI is an HTML-based browser application that is
+supported on the most recent three major versions of the following
+browsers:</span>
+
+<span class="c11"></span>
+
+-   <span class="c11">Chrome</span>
+-   <span class="c11">Firefox</span>
+-   <span class="c11">Safari</span>
+
+<span class="c29">Yavin installation guide</span>
+-------------------------------------------------
+
+The “<span class="c40">Installation Guide</span><span class="c11">” link
+takes you through the step-by-step guide in installing all the packages
+and files needed to have a full operational Yavin on your system.
+</span>
+
+<span class="c29">About data ingestion and data sources</span>
+--------------------------------------------------------------
+
+Before you can use Yavin to visualize and analyze data, you need to
+configure the Elide portion of Yavin to access the data you wish to
```

Comment:
Before you can use Yavin to visualize and analyze data, you need to define your data model and database configuration in Elide.